ui/tests/integration/components/known-secondaries-card-test.js

```diff
@@ -0,0 +1,65 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import engineResolverFor from 'ember-engines/test-support/engine-resolver-for';
+import hbs from 'htmlbars-inline-precompile';
+const resolver = engineResolverFor('replication');
+
+const CLUSTER = {
+  canAddSecondary: true,
+  replicationMode: 'dr',
+};
+
+const REPLICATION_ATTRS = {
+  knownSecondaries: ['firstSecondary', 'secondary-2', '3'],
+};
+
+module('Integration | Component | replication known-secondaries-card', function(hooks) {
+  setupRenderingTest(hooks, { resolver });
+
+  hooks.beforeEach(function() {
+    this.set('cluster', CLUSTER);
+    this.set('replicationAttrs', REPLICATION_ATTRS);
+  });
+
+  test('it renders with a table of known secondaries', async function(assert) {
+    await render(hbs`<KnownSecondariesCard @cluster={{cluster}} @replicationAttrs={{replicationAttrs}} />`);
+
+    assert
+      .dom('[data-test-known-secondaries-table]')
+      .exists('shows known secondaries table when there are known secondaries');
+    assert.dom('[data-test-manage-link]').exists('shows manage link');
```

Comment:
Can we add an assertion that the number of rows match the number of known secondaries here?